website/docs/maintenance/backup_restore.md

```diff
@@ -10,7 +10,7 @@ If you are using `docker-compose`, you are using Docker Compose v1, which has be
 Create backup file `teslamate.bck`:
 
 ```bash
-docker compose exec -T database pg_dump -U teslamate teslamate > /backuplocation/teslamate.bck
+docker compose exec -T database pg_dumpall -U teslamate > /backuplocation/teslamate.bck
```

Comment:
Please add test results for this, comparing db size, db entries and so on, or revert to pg_dump